### PR TITLE
[GMS-229] Small clarifying doc improvements

### PR DESCRIPTION
**Pull request by usamaSaddiq**
## Description

This PR adds some clarifying improvements to the docs about various things.

## Resolved issues
Jira: https://immutable.atlassian.net/jira/software/c/projects/GMS/boards/289?modal=detail&selectedIssue=GMS-229

### Before submitting the PR, please consider the following:
- [ ] It's beneficial if your pull request references an issue used to discuss the pull request ahead of time. If you haven't previously created an issue, please create one and discuss your contribution with the maintainers.
- [ ] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [ ] The description should clearly illustrate what problems the pull request solves.
- [ ] Ensure that the commit messages follow our guidelines.
- [ ] Resolve merge conflicts (if any).
- [ ] Make sure that the current branch is up to date with the `main` branch.

### For internal Immutable developers:
- [ ] If you are adding or updating a documentation page, please also add or update the team ownership of that page (follow [this guide](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1947927045/Docs+ownership+registry#What-to-do-when-adding%2Fupdating-documentation-pages%3F)).
- [ ] If you are adding or updating documentation for an SDK, please make sure that you meet the requirements in [this doc](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1916994017/SDK+documentation+guide).
- [ ] Please obtain PR approval from all of the following:
    - Your tech lead (Fallback: Team lead or engineering manager)
    - Your product manager (Fallback: Group PM)
    - Another software engineer on your team

### CODE REVIEW

File: docs/main/guides/integrate-your-application/personal-inventory.md [hunk 1]
```diff
@@ -21,13 +21,16 @@ ImmutableX’s base token is Ethereum. Once deposited onto the platform it exist
 - **Preparing withdrawal** - ETH that has started the withdrawal process from ImmutableX. It must wait until the next published signature to the main chain before moving to the withdrawable state.
 - **Withdrawable** - ETH that is now withdrawable to another address on the main chain.
 
-To retrieve a user’s balances use the following IMX client function:
+To retrieve a user’s balance, use the following IMX client function:
```
Comment (allan-almeida) [inline docs/main/guides/integrate-your-application/personal-inventory.md:24]: this is plural as [listBalances return an array](https://docs.x.immutable.com/reference#/operations/listBalances)

File: docs/main/guides/integrate-your-application/personal-inventory.md [hunk 2]
```diff
@@ -21,13 +21,16 @@ ImmutableX’s base token is Ethereum. Once deposited onto the platform it exist
...
 ```javascript
-const balances = await client.getBalances({ user: address })
+// The address is the value returned by the link.setup({}) call i.e
+// const { address, starkPublicKey } = await link.setup({});
+const listBalancesResponse = await client.listBalances({ user: address })
+const balances = listBalancesResponse.result[0];
 ```
 
-The _balances_ object will contain the amounts of all three states: imx, preparing_withdrawal, and withdrawable. Each value is returned as a BigNumber, a structure used to hold the many decimal places an ETH amount needs for display. The value is denominated in Wei, so a conversion may be necessary if trying to display the value as ETH. 1 ETH = 10^18 Wei.
+The _balances_ object will contain the amounts of all three states: balance(respresenting your balance in eth on the imx protocol), preparing_withdrawal, and withdrawable. Each value is returned as a BigNumber, a structure used to hold the many decimal places an ETH amount needs for display. The value is denominated in Wei, so a conversion may be necessary if trying to display the value as ETH. 1 ETH = 10^18 Wei.
```
[the remark this answers is not in the document]
Comment (allan-almeida) [reply docs/main/guides/integrate-your-application/personal-inventory.md:33]: not sure about `balance in eth` part, what if the symbol is for a non-eth denomination e.g. `IMX` or `GOG`?

[the remark this answers is not in the document]
Comment (usamaSaddiq) [reply docs/main/guides/integrate-your-application/personal-inventory.md:33]: Ok will remove 👍🏽

File: docs/main/guides/integrate-your-application/personal-inventory.md (hunk 1 above)
Comment (usamaSaddiq) [reply docs/main/guides/integrate-your-application/personal-inventory.md:24]: 👍🏽  will update

File: docs/main/guides/integrate-your-application/personal-inventory.md [hunk 3]
```diff
@@ -21,13 +21,16 @@ ImmutableX’s base token is Ethereum. Once deposited onto the platform it exist
...
 ```javascript
-const balances = await client.getBalances({ user: address })
+// The address is the value returned by the link.setup({}) call i.e
+// const { address, starkPublicKey } = await link.setup({});
+const listBalancesResponse = await client.listBalances({ user: address })
+const balances = listBalancesResponse.result[0];
 ```
 
-The _balances_ object will contain the amounts of all three states: imx, preparing_withdrawal, and withdrawable. Each value is returned as a BigNumber, a structure used to hold the many decimal places an ETH amount needs for display. The value is denominated in Wei, so a conversion may be necessary if trying to display the value as ETH. 1 ETH = 10^18 Wei.
+The _balances_ object will contain the amounts of all three states: balance(respresenting your balance on the imx protocol), preparing_withdrawal, and withdrawable. Each value is returned as a BigNumber, a structure used to hold the many decimal places an ETH amount needs for display. The value is denominated in Wei, so a conversion may be necessary if trying to display the value as ETH. 1 ETH = 10^18 Wei.
```
Comment (allan-almeida) [inline docs/main/guides/integrate-your-application/personal-inventory.md:33]: type `respresenting`

**Review by allan-almeida — APPROVED** [review]

**Pull request merged**